pkg/router/controller/status.go

```diff
@@ -144,13 +142,13 @@ func performIngressConditionUpdate(action string, lease writerlease.Lease, track
 		// value, skip updating altogether and rely on the next resync to resolve conflicts. This prevents routers
 		// with different configurations from endlessly updating the route status.
 		if !created && tracker.IsChangeContended(key, now, original) {
-			glog.V(4).Infof("%s: skipped update due to another process altering the route with a different ingress status value: %s %#v", action, key, original)
+			log.V(4).Info("skipped update due to another process altering the route with a different ingress status value", "action", action, "key", key, "original", original)
 			return writerlease.Release, false
 		}
 
 		switch _, err := oc.Routes(route.Namespace).UpdateStatus(route); {
 		case err == nil:
-			glog.V(4).Infof("%s: updated status of %s/%s", action, route.Namespace, route.Name)
+			log.V(4).Info("updated status", "action", action, "namespace", route.Namespace, "name", route.Name)
```

Comment:
"updated route status" would be clearer.

Reply:
Fixed